### PR TITLE
Add support for model image data

### PR DESCRIPTION
**Pull request by joeyballentine**
Used images directly from the esrgan repo for this

image

Also added support for non-paired images, since some people might have their own comparisons like this:

image

### CODE REVIEW

**Comment by RunDevelopment** [comment]
> Also added support for non-paired images, since some people might have their own comparisons

Do we even want to support those?

**Comment by joeyballentine** [comment]
I think we should. there's plenty of papers with their own ablation comparison images, and maybe people just wanna show off results without an LR.

**Comment by RunDevelopment** [comment]
Nah, you're right. I didn't think about research works.

**Comment by joeyballentine** [comment]
image
also added editing, adding, and reordering

image

**Comment by RunDevelopment** [comment]
1. image
2. The thumbnail shouldn't be stretched.

File: src/pages/models/[id].tsx [hunk 1]
```diff
@@ -64,11 +57,15 @@ export default function Page({ modelId, modelData }: Props) {
 
     const { updateModelProperty } = useUpdateModel(webApi, modelId);
 
+    const firstImageValue = model.images[0];
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (RunDevelopment) [inline src/pages/models/[id].tsx:61]:
```suggestion
    const firstImageValue = model.images[0] as Image | undefined;
```

**Pull request merged**